### PR TITLE
remove httpclient and use qiniu origin domain download

### PR DESCRIPTION
**Pull request by Boxie5**

### CODE REVIEW

File: src/main/java/com/aliyun/oss/internal/OSSObjectOperation.java [hunk 1]
```diff
@@ -269,32 +264,55 @@ public AppendObjectResult appendObject(AppendObjectRequest appendObjectRequest)
     public OSSObject getObject(GetObjectRequest getObjectRequest) throws OSSException, ClientException {
 
         return (getEndpoint().toString().indexOf("aliyuncs.com") >= 0) 
-                ? getObject_oss(getObjectRequest) : getObject_qiniu(getObjectRequest);
+                ? getOSSObject(getObjectRequest) : getQiniuObject(getObjectRequest);
     }
 
-    public OSSObject getObject_qiniu(GetObjectRequest getObjectRequest) throws OSSException, ClientException {
+    public OSSObject getQiniuObject(GetObjectRequest getObjectRequest) throws OSSException, ClientException {
+        getLog().info("==== getObject: " + getObjectRequest.getBucketName() + ":" + getObjectRequest.getKey());
```
Comment (ahguang973) [inline src/main/java/com/aliyun/oss/internal/OSSObjectOperation.java]: info -> debug

File: src/main/java/com/aliyun/oss/internal/QiniuRequestSigner.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+        if (accessKeyId.length() > 0 && secretAccessKey.length() > 0) {
+            // String url = String.format("http://", QiniuRequestSigner.DEFAULT_ORIGIN_DOMAIN, request.getAbsoluteUrl());
+            Auth auth = Auth.create(this.creds.getAccessKeyId(), this.creds.getSecretAccessKey());
+            try {
+                URL originURL = new URL("http", this.endPoint.toString(), this.key);
+                String privateUrl = auth.privateDownloadUrl(originURL.toString());
+                URL url = new URL(privateUrl);
+                url = new URL("http", QiniuRequestSigner.DEFAULT_ORIGIN_DOMAIN, url.getFile());
+                System.out.println("private url with token");
```
Comment (ahguang973) [inline src/main/java/com/aliyun/oss/internal/QiniuRequestSigner.java]: remove println

File: src/main/java/com/aliyun/oss/internal/OSSObjectOperation.java (hunk 1 above)
Comment (Boxie5) [reply src/main/java/com/aliyun/oss/internal/OSSObjectOperation.java]: fixed

File: src/main/java/com/aliyun/oss/internal/QiniuRequestSigner.java (hunk 2 above)
Comment (Boxie5) [reply src/main/java/com/aliyun/oss/internal/QiniuRequestSigner.java]: fixed

**Pull request merged**